acceptancetests/jujupy/wait_condition.py

```diff
@@ -227,6 +227,38 @@ def do_raise(self, model_name, status):
                         self.machine)
 
 
+class WaitMachinesNotPresent(BaseCondition):
```

Comment:
I'd prefer we just update the other condition to take one or more machines.

Reply:
I'm not sure about this, I think it's preferable if the wait conditions just did one thing?

Reply:
As a python dev I'd say it is doing one thing. Waiting for machines to go away. If you're doing one or 3 it's the same job with the same checks and the same process.